sdk/servicebus/service-bus/tests.yml

```diff
@@ -26,7 +27,19 @@ jobs:
         AAD_CLIENT_ID: $(aad-azure-sdk-test-client-id)
         AAD_TENANT_ID: $(aad-azure-sdk-test-tenant-id)
         AAD_CLIENT_SECRET: $(aad-azure-sdk-test-client-secret)
-        AZURE_SUBSCRIPTION_ID: $(test-subscription-id)
-        CLEAN_NAMESPACE: "true"
-        RESOURCE_GROUP: $(service-bus-test-resource-group)
-        SERVICEBUS_CONNECTION_STRING: $(service-bus-test-connection-string)
+        AZURE_SUBSCRIPTION_ID: $(AZURE_SUBSCRIPTION_ID)
+        SERVICEBUS_CONNECTION_STRING: $(SERVICEBUS_CONNECTION_STRING)
+        RESOURCE_GROUP: $(RESOURCE_GROUP)
+        QUEUE_NAME: "partitioned-queue"
+        QUEUE_NAME_NO_PARTITION: "unpartitioned-queue"
+        QUEUE_NAME_SESSION: "partitioned-sessions-queue"
+        QUEUE_NAME_NO_PARTITION_SESSION: "unpartitioned-sessions-queue"
+        TOPIC_NAME: "performance-test-topic"
+        TOPIC_NAME_NO_PARTITION: "unpartitioned-topic"
+        TOPIC_NAME_SESSION: "partitioned-sessions-topic"
+        SUBSCRIPTION_NAME: "performance-test-2"
```

Comment:
Please update this to "partitioned-subscription" (if this must be specified, for say, creating this upfront).

Currently Service Bus requires a second duplicate set for browser tests.
And another different set of resources like this one for the atom management API tests (and counterpart for browser)

Will this PR eventually concat a unique GUID to the resource names being created?

Reply:
these resources will always be deleted after the tests finish

Reply:
@sadasant The question was whether the queues/topics/subscriptions being created by the arm template will have unique names each time

Reply:
no, they will have deterministic names, which are the ones defined here. We can also remove these names and let the underlying code decide.

Reply:
On other clients, the uniqueness of the name is done in TypeScript directly. It might be obvious already, but just to make sure it's stated nearby, keep in mind that this whole resource where each topic, subscription, etc, are created, will be re-generated every time.